001-yii-values.md

```diff
@@ -2,75 +2,70 @@
 
 ## Goal
 
-The Yii project goal is to build and maintain:
-
-- Practical and helpful tools and community.
+The Yii project aims to build and maintain _practical_ and _helpful_ tools and community.
 
 ## Values
 
-We share some common values that serve the goal.
+The values we express in our work support the goal. We try to
 
-- Being Practical
+- Be Practical
```

Comment:
I set the things in _imperative voice_ where it made sense.

Btw, I also think it's the best way to write code descriptions and comments. It remove the subject of the sentence. The comment/docblock description states in the form of a command and the following code will comply.

```php
/** Return the subsequent Foo for the given Bar */
function getFollowingFoo(Bar $bar): Foo
{}
```

(Btw: If you write a good description and choose good names for function and params, and use type hints, `@param` and `@return`  tags often become redundant. So remove them to be DRY.)